cmd/run.go

```diff
@@ -18,6 +18,7 @@ import (
 	"fmt"
```

Comment:
Change `RunCmd.Flags().StringVarP(&runFilePath, "run-file", "f", "", "Path to the configuration file for the apps to run")`
```suggestion
	RunCmd.Flags().StringVarP(&runFilePath, "run-file", "f", "", "Path to the run template file for the list of apps to run")
```